ajax/libs/datalist-polyfill/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "datalist-polyfill",
+  "description": "A minimal and dependency-free vanilla JavaScript datalist polyfill. Supports all standard's functionality as well as mimics other browsers behavior.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mfranzke/datalist-polyfill.git"
+  },
+  "author": "Maximilian Franzke",
+  "license": "MIT",
+  "filename": "datalist-polyfill.min.js",
+  "keywords": [
+    "datalist",
+    "polyfill",
+    "javascript",
+    "html",
+    "autosuggest",
+    "autosuggester",
+    "suggest",
+    "suggester",
+    "select"
+  ],
+  "npmName": "datalist-polyfill",
+  "npmFileMap": [
```

Comment:
I do hope that these are filled out correctly - it's mainly the two JS files that would need to get hosted on a CDN for providing those to the users. And those are stored in the root of the project right away.